jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -101,14 +101,22 @@ public BaseImageWithAuthorization call()
           RegistryAuthenticationFailedException {
     buildConfiguration
         .getBuildLogger()
-        .lifecycle("Getting base image " + buildConfiguration.getBaseImageReference() + "...");
+        .lifecycle(
+            "Getting base image "
+                + buildConfiguration.getBaseImageReference()
+                + " with no auth...");
```

Comment:
Since the default case is `gcr.io/distroless/java`, I think we can leave off the `with no auth` here, and then below it can be what @chanseokoh recommended.